maven/bnd-reporter-maven-plugin/src/test/resources/integration-test/test/report-multi-project/pom.xml

```diff
@@ -0,0 +1,36 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>biz.aQute.bnd-test</groupId>
+		<artifactId>reporter-test</artifactId>
+		<version>0.0.1</version>
+	</parent>
+
+	<artifactId>report-multi-project</artifactId>
```

Comment:
You should add `<groupId>biz.aQute.bnd-test</groupId>` to the coordinates.

Reply:
Why should I explicitly add  the GroupId, it is not inherited from the parent?

Reply:
For consistency with all other poms and to avoid defaulting. Lets be explicit.